src/HomeComponent/HomeComponent.js

```diff
@@ -1,30 +1,27 @@
+/* eslint-disable eqeqeq */
 import React from 'react';
 import './HomeComponent.css';
 import { Link } from 'react-router-dom';
 import { getObject } from '../exerciseStorageService';
 
 function HomeComponent () {
-  function weekStatus (n) {
+  function getWeekStatus (weekNumber) {
     const exerciseData = getObject();
-    const weekOneDayOneIsFinished = !!(exerciseData.finished.find(x => x[0] == n && x[1] == 1));
-    const weekOneDayTwoIsFinished = !!(exerciseData.finished.find(x => x[0] == n && x[1] == 2));
-    const weekOneDayThreeIsFinished = !!(exerciseData.finished.find(x => x[0] == n && x[1] == 3));
+    const isDayOneFinished = !!(exerciseData.finished.find(x => x[0] == weekNumber && x[1] == 1));
+    const isDayTwoFinished = !!(exerciseData.finished.find(x => x[0] == weekNumber && x[1] == 2));
+    const isDayThreeFinished = !!(exerciseData.finished.find(x => x[0] == weekNumber && x[1] == 3));
 
-    let weekOneStatus = 0;
-    if (weekOneDayOneIsFinished) {
-      weekOneStatus = 1;
-      return weekOneStatus;
+    if (isDayOneFinished) {
+      return 1;
     }
-    if (weekOneDayTwoIsFinished) {
-      weekOneStatus = 2;
-      return weekOneStatus;
+    if (isDayTwoFinished) {
+      return 2;
     }
 
-    if (weekOneDayThreeIsFinished) {
-      weekOneStatus = 3;
-      return weekOneStatus;
+    if (isDayThreeFinished) {
+      return 3;
     }
-    if (!weekOneDayOneIsFinished && !weekOneDayOneIsFinished && !weekOneDayThreeIsFinished) {
+    if (!isDayOneFinished && !isDayTwoFinished && !isDayThreeFinished) {
```

Comment:
you don't need this check here, since if either of those was true the function would have already returned. Remove this if statement and simply return 0.